pkg/client/deploymentlogs.go

```diff
@@ -34,15 +35,27 @@ func newDeploymentLogs(c *Client, namespace string) *deploymentLogs {
 
 // Get gets the deploymentlogs and return a deploymentLog request
 func (c *deploymentLogs) Get(name string, opts api.DeploymentLogOptions) *kclient.Request {
-	req := c.r.Get().Namespace(c.ns).Resource("deploymentConfigs").Name(name).SubResource("log")
+	req := c.r.Get().Namespace(c.ns).Resource("deploymentConfigs").Name(name).SubResource("log").
+		Param("follow", strconv.FormatBool(opts.Follow)).
+		Param("timestamps", strconv.FormatBool(opts.Timestamps))
```

Comment:
This looks really fragile.  Is there not a way to automatically do this based on tags?